.github/workflows/release.yml

```diff
@@ -99,7 +100,7 @@ jobs:
         with:
           ignore_install_scripts: true
 
-      - run: npm publish --provenance
+      - run: npm publish --provenance --tag alpha
```

Comment:
this ensures that any alpha releases I run are released as alphas.  we will revert this before the official 2.0 release